README.md

```diff
@@ -47,7 +47,7 @@ File           | Description
 The code in `index.js` calls the Wikipedia API and retrieves an article. In order to do that, it needs the Axios library. Axios is already listed in `package.json`, so you can install it with:
 
 ```console
-npm ci
+npm install
```

Comment:
Any particular reason you prefer `npm install` over `npm ci`? I thought `npm ci` was best practice, because it ensures the user has the exact same packages as were used when creating the tutorial. Right now, all tutorials use `npm ci`, but if there’s a good reason to prefer `npm install`, I’ll update the other tutorials as well.

I did notice package-lock.json was in .gitignore. That’s probably a mistake.

Reply:
only did it because it was failing when i tried locally and npm could not find the lock file. i think we just have to be careful to not include any package from our private NPM registry (under hostname `npm.pkg.github.com`). then push the lock file and change to `npm ci`. do you agree?

Reply:
I made the necessary changes. The private NPM registry shouldn’t be a problem, because the CLI doesn’t come pre-installed; it’s a step in the tutorial. :-)

Reply:
nice. thanks!